chatterbot_corpus/corpus.py

```diff
@@ -88,9 +88,10 @@ def load_corpus(self, dotted_path):
         for file_path in data_file_paths:
             corpus = self.read_corpus(file_path)
 
-            corpora.categories = corpus.get('categories', [])
-            conversations = corpus.get('conversations', [])
+            categories = corpus.get('categories', [])
+            corpora.categories.extend([categories])
```

Comment:
Although the order of the categories may be possible to track, I think there might be a slightly more straightforward way to accomplish this. I'm going to open up a pull request with an alternative approach. I would appreciate it if you can confirm that it achieves the same goal as these changes.